setup.cfg

```diff
@@ -69,7 +69,7 @@ where = src
 
 [options.extras_require]
 ray =
-    ray[default]==2.0.1
+    ray[default]==2.3.0
     # Ray 2.0.0 requires pyarrow but it's missing from their requirements.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Can you check this too?

Reply:
You mean pyarrow dependency?

Reply:
Ok I can see they have this dependency on their list: 
https://github.com/ray-project/ray/blob/master/python/requirements.txt#L62
Im gonna remove ours (unless we use pyarrow somewhere in our project in different places)

Reply:
Check that this is included when you `pip install ray[default]`

Reply:
Good point - so it doesnt.
According to this issue: https://github.com/ray-project/ray/issues/25952
We should be using ray[data] as dependency as well, but it brings things like pandas, which im not certain we want. I'd leave pyarrow here only.